grammars.yml

```
@@ -703,6 +703,11 @@ vendor/grammars/latex.tmbundle:
 vendor/grammars/linter-lilypond:
 - source.lilypond
 vendor/grammars/liquid-tm-grammar:
+- liquid.injection
+- source.css
+- source.js
+- source.json
+- text.html.basic
 - text.html.liquid
```

Remove these added lines, please. The only entry in this list should be `text.html.liquid`:

```suggestion
- text.html.liquid
```

---

I removed these manually as the extra commit will be squashed anyways.